sui-execution/src/v0.rs

```diff
@@ -193,4 +195,26 @@ impl<'m> verifier::Verifier for Verifier<'m> {
             self.metrics,
         )
     }
+
+    fn meter_compiled_modules_with_overrides(
+        &mut self,
+        modules: &[CompiledModule],
+        protocol_config: &ProtocolConfig,
+        config_overrides: &mut VerifierOverrides,
+    ) -> SuiResult<()> {
+        config_overrides.max_per_fun_meter_current = self.config.max_per_fun_meter_units;
+        config_overrides.max_per_mod_meter_current = self.config.max_per_mod_meter_units;
+        self.config.max_per_fun_meter_units = config_overrides.max_per_fun_meter_units;
+        self.config.max_per_mod_meter_units = config_overrides.max_per_mod_meter_units;
+        run_metered_move_bytecode_verifier(
+            modules,
+            protocol_config,
+            &self.config,
+            &mut self.meter,
+            self.metrics,
+        )?;
+        config_overrides.fun_meter_units_result = self.meter.get_usage(Scope::Function);
+        config_overrides.mod_meter_units_result = self.meter.get_usage(Scope::Function);
```

Comment:
The interface of mutating the overrides is a little funky -- could we not split overrides into a read-only part and a return part?

Reply:
we could and it would be a bit more plumbing. Let me try that and see how it looks.
To be honest I am not too clear we need this whole story and it was mostly to minimize changes.
This is just to get a specific command in the CLI working and don't know....